src/custom/constants/lists.ts

```diff
@@ -27,6 +27,9 @@ const BA_LIST = 'https://raw.githubusercontent.com/The-Blockchain-Association/se
 // Rinkeby Default
 const RINKEBY_LIST = RAW_CODE_LINK + '/main/src/custom/tokens/rinkeby-token-list.json'
 
+// Rinkeby Default
+const GOERLI_LIST = RAW_CODE_LINK + '/main/src/custom/tokens/goerli-token-list.json'
```

Comment:
This file doesn't exist.
Add a TODO?

Reply:
Added a PR for this https://github.com/cowprotocol/cowswap/pull/569